README.md

```diff
@@ -10,6 +10,76 @@ It can be used by installing this package & setting the `extends` property in yo
 }
 ```
 
+Alternative yml config  `.eslintrc.yml`:
+```yml
+---
+  extends: "@dosomething/eslint-config"
+```
+
+### Additional Profiles
+
+This package includes additional linting profiles fine-tuned to different environments.
+
+**Node.js 8.x**
+
+Provides ESLint config specific to native (non-transcripted, non-babel) Node.js 8.x ECMA 2017 apps.
```

Comment:
Yes, poor choice of word. I meant "[transpiled](https://en.wikipedia.org/wiki/Source-to-source_compiler)".